source/NVDAObjects/UIA/__init__.py

```diff
@@ -858,6 +858,10 @@ def findOverlayClasses(self,clsList):
 		except ValueError:
 			pass
 
+		# Support Windows Console's UIA interface
+		if self.windowClassName == "ConsoleWindowClass" and self.UIAElement.cachedAutomationId == "Text Area" and config.conf['UIA']['consoleUIA']:

```

Comment:
I guess the config check is not necessary here, as I think the badUIAWindowClassNames check already deals with this.